syntax = "proto3";

service ServicioEjemplo {
  rpc Saludar (SolicitudSaludo) returns (RespuestaSaludo) {}
  rpc Sumar (SolicitudSuma) returns (RespuestaSuma) {}
}

message SolicitudSaludo {
  string nombre = 1;
}

message RespuestaSaludo {
  string mensaje = 1;
}

message SolicitudSuma {
  int32 a = 1;
  int32 b = 2;
}

message RespuestaSuma {
  int32 resultado = 1;
}